glue/plugins/wcs_autolinking/tests/test_wcs_autolinking.py

```diff
@@ -240,3 +241,254 @@ def test_celestial_with_unknown_axes():
     assert link[2].get_from_ids() == [px2, pz2]
     assert link[3].get_to_id() == py1
     assert link[3].get_from_ids() == [px2, pz2]
+
+
+def test_wcs_autolinking_of_2d_cube_with_temporal_and_spectral_axes_case_1():
+    """
+    A test to confirm that two 2D data cubes with matching number of dimensions
+    where the first is temporal and the next one spectral (vacuum wavelength in this
+    case) is indeed autolinked.
+    """
+
+    wcs1 = WCS(naxis=2)
+    wcs1.wcs.ctype = 'TIME', 'WAVE'
+    wcs1.wcs.set()
+
+    data1 = Data(label='Data 1')
+    data1.coords = wcs1
+    data1['x'] = np.ones((2, 3))
+    py1, px1 = data1.pixel_component_ids
+
+    wcs2 = WCS(naxis=2)
+    wcs2.wcs.ctype = 'TIME', 'WAVE'
```

Comment:
We could make this a little more general by changing TIME to e.g. TAI in wcs2 - this should still work but at least shows we aren't using matching by CTYPE string.

Reply:
Sure, also double-checked that it worked